admin/spec/service/marking.service.spec.js

```diff
@@ -1,34 +1,21 @@
 'use strict'
-/* global describe, beforeEach, afterEach, it, expect */
-const sinon = require('sinon')
-const proxyquire = require('proxyquire').noCallThru()
-
+/* global describe, beforeEach, afterEach, it, expect, spyOn */
 const checkDataService = require('../../services/data-access/check.data.service')
 const completedCheckDataService = require('../../services/data-access/completed-check.data.service')
 const completedCheckMock = require('../mocks/completed-check')
 
 describe('markingService', () => {
-  let service, sandbox
+  let service = require('../../services/marking.service')
 
   beforeEach(() => {
-    sandbox = sinon.sandbox.create()
+    // sandbox = sinon.sandbox.create()
```

Comment:
I am guessing you can just delete the sandbox creation and restoration right?

Reply:
yep, probably just forgot to remove it once i got the tests passing